src/services/user.service.js

```diff
@@ -0,0 +1,45 @@
+const { User } = require('../models/User.model.js');
+
+const getUsers = async () => {
+  const users = await User.findAll();
+
+  return users;
+};
+
+const createUser = (name) => {
+  return User.create({ name });
```

Comment:
Consider adding validation to ensure that the `name` is not an empty string before creating a user.